python/lsst/ip/diffim/dipoleFitTask.py

```diff
@@ -88,6 +88,29 @@ class DipoleFitPluginConfig(measBase.SingleFramePluginConfig):
         (note this is actually a significance, not a chi2 value).""")
 
 
+class DipoleFitTaskConfig(measBase.SingleFrameMeasurementConfig):
+    """!Measurement of detected diaSources as dipoles"""
+
+    def setDefaults(self):
+        measBase.SingleFrameMeasurementConfig.setDefaults(self)
+
+        self.plugins.names = ["base_CircularApertureFlux",
+                              "base_PixelFlags",
+                              "base_SkyCoord",
+                              "base_PsfFlux",
+                              "ip_diffim_NaiveDipoleCentroid",
+                              "ip_diffim_NaiveDipoleFlux",
+                              "ip_diffim_PsfDipoleFlux"
+                              ]
+
+        self.slots.calibFlux = None
+        self.slots.modelFlux = None
```

Comment:
That is not the current behavior. PsfDipoleFlux does not set an overall flux (just separate fluxes for pos. and neg. lobes). I would set it to the flux from ip_diffim_DipoleFit, but I am not sure how to, since that task is not formally "registered" as a plugin (this will be fixed in a future ticket after the sfm overhaul).